pages/index.tsx

```diff
@@ -28,15 +28,15 @@ export async function getServerSideProps(context): Promise<{props: SSProps}> {
 
 const chakraUiRendererTheme: Parameters<typeof ChakraUIRenderer>[0] = {
   a: (props) => {
-    const { children } = props;
-    const link = children[0] as string;
+    const { children, href } = props;
+
     return (
-      <Link target="_blank" rel="noopener noreferrer" href={link} color="#4987BD">
+      <Link target="_blank" rel="noopener noreferrer" href={href} color="#4987BD">
         {children}
       </Link>
     );
   },
-};
```

Comment:
We need to get eslint set up so we're not changing a bunch of lines unnecessarily between our different code settings